vars/publishIntegTestResults.groovy

```diff
@@ -54,25 +54,32 @@ void call(Map args = [:]) {
 
     manifest.components.each { component ->
         def componentName = component.name
+        def componentRepo = component.repository.split('/')[-1].replace('.git', '')
+        def componentRepoUrl = component.repository.replace("https://github.com/", "github.com/").replace(".git", "")
```

Comment:
Can we just do a substring so no matter what people put before `github.com` we are good?

def componentRepoUrl = component.repository.substring(component.repository.indexOf("github.com")).replace(".git", "")

def componentRepoUrl =

Reply:
Thanks updated Peter.